src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTestBase.cs

```diff
@@ -77,9 +60,12 @@ protected static HttpClientHandler CreateHttpClientHandler(bool useSocketsHttpHa
             return handler;
         }
 
+        // Manicka: remove
```

Comment:
Is this a TODO item for another PR you're doing?

Reply:
Removed, thanks for noticing.